evm/src/main/java/org/hyperledger/besu/evm/precompile/ECRECPrecompiledContract.java

```diff
@@ -76,13 +80,13 @@ public long gasRequirement(final Bytes input) {
   public PrecompileContractResult computePrecompile(
       final Bytes input, @NotNull final MessageFrame messageFrame) {
     final int size = input.size();
-    final Bytes d = size >= 128 ? input : Bytes.wrap(input, MutableBytes.create(128 - size));
-    final Bytes32 h = Bytes32.wrap(d, 0);
```

Comment:
this second wrap to Bytes32 was just not needed?

Reply:
It's moved both inside computeK1Native and computeDefault, also renamed from h -> messageHash

I could have calculated outside and passed it in but thought the unary method signature of computeX(safeInput) was clearer than computeX(safeInput, messageHash)